### DISCUSSION TITLE
Theme choose

### DISCUSSION THREAD

**Discussion by Bruno-serpa** [Sketchware-Pro/Sketchware-Pro#1165] [Ideas]
I think it would be good add an option to be able to choose whether to leave sketchware in light or dark mode, regardless of the mode your cell phone is in.

**Comment by thisisshadow** [comment]
I will work on such a feature, bro, but I don't know when it will happen, approximately within a week.

**Reply by thisisshadow** [reply]
> I will work on such a feature, bro, but I don't know when it will happen, approximately within a week.

But it will only be valid in material sketchware pro

**Comment by trindadedev13** [comment]
https://github.com/Sketchware-Pro/Sketchware-Pro/pull/1178
here brother, now just wait to be merged
